components/datetime/tests/fixtures/tests/lengths.json

```diff
@@ -95,13 +65,13 @@
             "value": "2020-03-21T08:25:07.000",
             "options": {
                 "length": {
-                    "time": "full",
-                    "date": "full"
+                    "time": "short",
+                    "date": "medium"
```

Comment:
**Author Clarification**

`DateTimeFormat` no longer supports patterns that contain time-zone symbols, and returns an `Err` on construction. This means it can no longer be used with `Time::Long` and `Time::Full`. 

These are tested by `ZonedDateTimeFormat` which pulls from `lengths_with_zones.json`.